From 25b71205ec7a762c67e2d80f67fa03b80bd789b9 Mon Sep 17 00:00:00 2001
From: Darren Etheridge <detheridge@ti.com>
Date: Thu, 16 Jan 2014 14:46:01 -0600
Subject: [PATCH 154/285] ARM: dts: am437x-gp-evm remove duplicate tscadc node

Remove duplicated node - no impact other than makes
dts cleaner.

Signed-off-by: Darren Etheridge <detheridge@ti.com>
[Kevin: The original patch taken from TI
ti-sdk-am437x-evm-07.01.00.00-Linux-x86-Install.bin]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm/boot/dts/am437x-gp-evm.dts |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/arch/arm/boot/dts/am437x-gp-evm.dts b/arch/arm/boot/dts/am437x-gp-evm.dts
index 10daba4..99343c7 100644
--- a/arch/arm/boot/dts/am437x-gp-evm.dts
+++ b/arch/arm/boot/dts/am437x-gp-evm.dts
@@ -469,14 +469,6 @@
 	};
 };
 
-&tscadc {
-	status = "okay";
-
-	adc {
-		ti,adc-channels = <0 1 2 3 4 5 6 7>;
-	};
-};
-
 &ecap0 {
 	status = "okay";
 	pinctrl-names = "default";
-- 
1.7.5.4

